consumer/README.md

```diff
@@ -6,19 +6,34 @@ operated by a third party.
 ## Requirements
 
 * Access to a kcp workspace.
-* Path of the kcp workspace in which the service is deployed (e.g. `root:pipeline-service`).
-This path is provided by the service operators.
+* The following information from the service provider:
+  * Path of the kcp workspace in which the service is deployed (e.g. `root:pipeline-service`).
+  * Name of the APIExport (e.g. `kubernetes`).
 
-## How to bind to the service
+## Quick start guide
 
 * Login on your kcp workspace.
-* Run `./bind_to.sh root:path:to:the:service` to bind to a Pipeline Service provider. The
-exact path will be provided by the service provider.
+* Run `./bind.sh --from root:path:to:the:service --to '~:pipeline-service'` to bind to a Pipeline Service provider.
+  * See the `--help` of the command for additional options (e.g. specifying the kubeconfig file).
+* A test will automatically run to validate that `pipelineruns` get scheduled.
+
+## How to bind to the service
```

Comment:
It is not clear enough that this section "How to bind to the service" and the above section "Quick start guide" are one and the same. To me, they look like steps to be followed one after another. Could you please make it clear to the user that they either need to run the script **or** need to do this step and the next step manually.

Reply:
Updated. Let me know if further changes are needed.